packages/main/src/themes/PopupsCommon.css

```diff
@@ -85,3 +81,47 @@
 :host([disable-scrolling]) .ui5-popup-content {
     overflow: hidden;
 }
+
+/*** Responsive paddings for the content***/
+:host([media-range="S"]) .ui5-popup-content {
+	padding: 1rem var(--_ui5_popup_content_padding_s);
```

Comment:
These responsive paddings for the content are defined as:
Size S: 0 1rem;
Size M: 0 2rem;
Size L: 0 2rem;
Size XL: 0 3rem;

Remove the top and bottom paddings.

Reply:
Why to remove top and bottom paddings?
See: B2 paddings here https://wiki.wdf.sap.corp/wiki/pages/viewpage.action?pageId=2033847432

Reply:
Ok, this is fine then. Keep the top and bottom padding.